test_shopcart.py

```diff
@@ -29,6 +29,175 @@ def test_index(self):
         self.assertTrue ('Shopcart Demo REST API Service' in resp.data)
 
 
+    def test_create_shopcarts(self):
```

Comment:
Ya I can do that!! Does everyone agree on this? @WesPainter @jiweix @ceciliazhang12